website/docs/specification/index.md

```diff
@@ -7,45 +7,37 @@ import Disclaimer from '../\_disclaimer.mdx';
+## Specifications
 
-## Digital Product Passport (DPP)
+UNTP is broken into several distinct and separately implementable components so that each actor can implement only what is relevant for their role. 
 
-The digital product passport (DPP) is issued by the shipper of goods and is the carrier of **product and sustainability information** for every serialised product item (or product batch) that is shipped between actors in the value chain. It is deliberately **simple and lightweight** and is designed to carry the minimum necessary data at the **granularity** needed by the receiver of goods - such as the scope 3 emissions in a product shipment. The passport contains links to **conformity credentials** which add trust to the ESG claims in the passport. The passport also contains links to **traceability events** which provide the "glue" to follow the Linked Data trail (subject to confidentiality constraints) from finished product back to raw materials. The UNTP DPP does not conflict with national regulations such as the EU DPP.  In fact, it can usefully be conceptualised as the **upstream B2B feedstock** that provides the data and evidence needed for the issuing of high quality national or regional level product passports.
+* The [Sustainability Vocabulary Catalog](SustainabilityVocabularyCatalog.md) is for scheme owners and regulators to publish their rules and criteria so that they can be unambiguously referenced by conformity claims about products or facilities.
+* The [Identity Resolver](IdentityResolver.md) and [Digital Identity Anchor](DigitalIdentityAnchor.md) are for operators of authoritative registers of businesses, facilities, trademarks, and products so that their members can prove their idenitty and link their identifiers to rich data like products passports and facility records.  
+* The [Digital Product Passport](DigitalProductPassport.md) and [Digital Facility Record](DigitalFacilityRecord.md) is for supply chain actors to securely publish discoverable imformation about their products and facilities, including sustainability claims.
```

Comment:
Small typo: s/is/are . There's also a split-infinitive in there ("to securely publish") if you care about that kind of thing :)